src/main/java/com/cmsc495/ticketsystem/config/SecurityConfig.java

```diff
@@ -1,47 +1,105 @@
 package com.cmsc495.ticketsystem.config;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.Customizer;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.core.userdetails.User;
-import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.security.core.userdetails.UserDetailsService;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.security.provisioning.InMemoryUserDetailsManager;
 import org.springframework.security.web.SecurityFilterChain;
+import com.cmsc495.ticketsystem.service.MyUserDetailService;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 
 @Configuration
+@EnableWebSecurity
 public class SecurityConfig {
 
+        @Autowired
+        private MyUserDetailService userDetailService;
+
+        @Autowired
+        private PasswordEncoder passwordEncoder;
+    /*
+     * @Bean
```

Comment:
Before the PR is merged we want to remove these comments

Reply:
i can't tell which comments for sure but if it is the unused code in securityConfig file i removed that